cluster/examples/kubernetes/ceph/common.yaml

```
@@ -373,9 +373,17 @@ subjects:
 # namespace. If you want to create the cluster in a different namespace, you will need to modify these roles
 # and bindings accordingly.
 #################################################################################################################
-# Service account for the Ceph OSDs. Must exist and cannot be renamed.
+# OLM: BEGIN SERVICEACCOUNT DEFAULT
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-ceph-default
+  namespace: rook-ceph
```

```suggestion
  namespace: rook-ceph # namespace:operator
```
the tests are failing because it was no replaced according to suites namespaces

---

Thanks @subhamkrai :+1: , the changes pointed above worked for me :)